keps/993-two-phase-admission/README.md

```diff
@@ -187,6 +188,34 @@ type AdmissionCheckParametersReference struct {
 	// Name is the name of the resource being referenced.
 	Name string `json:"name"`
 }
+
+// Configuration is the Schema for the kueueconfigurations API
+type Configuration struct {
+	...
+	// provisioningRequestRetryStrategy defines strategy for retrying ProvisioningRequest
+	ProvisioningRequestRetryStrategy *AdmissionCheckRetryStrategy `json:admissionCheckRetryStrategy, omitempty`
+}
+
+type ProvisioningRequestRetryStrategy struct {
+	// defaultRequeuingStrategy defines default requeueing strategy for ProvisioningRequests.
+	// The strategy is applied to all classes of ProvisioningRequest.
+	//
+	// Default values are:
+	//
+	// defaultRequeuingStrategyAC:
+	//	timestamp: Eviction
+	//	backoffLimitCount: 3
+	//	backoffBaseSeconds: 60
+	//	backoffMaxSeconds: 1800
+	DefaultRequeuingStrategy *RequeuingStrategy `json:defaultRequeuingStrategy`
+
+	// releaseQuota defines whether the reserved quota should be released during retry.
+	// If the releaseQuota is set not false, controller will delete old ProvisioningRequest
+	// and create a new one without releasing quota.
```

Comment:
```suggestion
	// releaseQuota defines whether the reserved quota should be released during when its ProvisioningRequest  fails.
	// If the releaseQuota is set not false, controller will delete old ProvisioningRequest
	// and create a new one without setting the workload's adission check state to `Retry`.
```

If an admission check state is set to `Retry` the workload controller will release the quota.